src/Schema.js

```diff
@@ -59,6 +59,33 @@ function getMergedTypeFieldsWithInterfaces(schema, type) {
 	return _.mergeWith(defaultFields, type.fields, mergeFields);
 }
 
+function collectSingleInterfaceDependencies(schema, interfaceName) {
+	const dependencies = [];
+	const processQueue = [interfaceName];
+
+	while (processQueue.length) {
+		const top = processQueue.shift();
+		const _interface = schema.interfaces[top];
+
+		if (_interface === undefined) throw new Error(`Interface "${top}" is not defined`);
```

Comment:
I just realised why we shouldn't throw if we don't find an interface. If we are declaring multiple schemas: `admin`, `public` then an interface might only exist in one of them but the type will extend it still

Reply:
Maybe we should also check that condition? If `interface.schema.includes(schema.name)`?